packages/api/core/src/api/make.ts

```diff
@@ -294,7 +292,7 @@ export const listrMake = (
                         arch: targetArch,
                       });
                     } catch (err) {
-                      if (err) {
```

Comment:
This is semantically different, in some places we throw strings, we should still somehow log those?

Reply:
Are there specific places where we throw strings? This was caught in lint by [`"@typescript-eslint/only-throw-error"`](https://typescript-eslint.io/rules/only-throw-error/).

Reply:
I've definitely done it before, probably in forge, or it's dependencies 😅

Reply:
Not sure if we still do it today or not

Reply:
Added another `else if` clause for when the error is a string as well: [12efc76](https://github.com/electron/forge/pull/3790/commits/12efc76108c12c06e4c163ee4eb20fc9e9ac7c37)